fn_ip_void/fn_ip_void/util/selftest.py

```diff
@@ -0,0 +1,45 @@
+# (c) Copyright IBM Corp. 2010, 2019. All Rights Reserved.
+# -*- coding: utf-8 -*-
+# pragma pylint: disable=unused-argument, no-self-use
+"""Function implementation
+   test with: resilient-circuits selftest -l fn-ip-void
+"""
+
+import logging
+import requests
+from fn_ip_void.util.ipvoid_helper import get_config_option
+
+log = logging.getLogger(__name__)
+log.setLevel(logging.INFO)
+log.addHandler(logging.StreamHandler())
+
+SUB_URL = "iprep/v1/pay-as-you-go/?key={}&stats"
```

Comment:
This is also in the helper, can we use it from there and remove this?

Reply:
Done!